src/Groups/sub.jl

```diff
@@ -175,13 +188,13 @@ julia> normal_subgroups(symmetric_group(5))
  Permutation group of degree 5 and order 1
 
 julia> normal_subgroups(quaternion_group(8))
-6-element Vector{PcGroup}:
- Pc group of order 8
- Pc group of order 4
- Pc group of order 4
- Pc group of order 4
- Pc group of order 2
- Pc group of order 1
+6-element Vector{SubPcGroup}:
+ Subgroup of pc group of order 8
+ Subgroup of pc group of order 4
+ Subgroup of pc group of order 4
```

Comment:
Hmmm... this is ambiguous, isn't it? The `4` refers to the order of the subgroup, but a reader might reasonably interpret this as saying the parent group has order 4... :-/.

(How) can we make this clearer? Perhaps We should call it a `Sub-pc group` (and explain that somewhere)? 

Or use a different phrasing: `Subgroup of pc group, has order 4` -- alas that would be different from how this kind of info is printed for almost anything else in OSCAR, wouldn't it?

Perhaps @thofma or @fieker have some brilliant insight on this.

Reply:
To be clear: we can address this in a follow-up PR (but then let's make an issue about this after the PR is merged so so we don't forget)

Reply:
I have opened issue  #3732.